### PR TITLE
Update to PSPDFKit 3.3.0

### LINKED ISSUE #20: Update React Native Wrapper for PSPDFKit 3.3 for Android
Since PSPDFKit for Android 3.3 is now available we need to check compatibility of our wrappers, and bump/update all versions.

### PR DESCRIPTION
**Pull request by simoarpe**
## Resolves #20 

Update to PSPDFKit 3.3.0

### CODE REVIEW

**Comment by radazzouz** [comment]
LGTM 👍

**Comment by radazzouz** [comment]
We could also bump the version to 1.3.1 (see #19) and tag as a new release.

**Comment by juliangrosshauser** [comment]
Does the iOS part still work after the `react-native` update?

**Comment by radazzouz** [comment]
I stand by my initial approval! 👍

File: samples/Catalog/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "Catalog",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (juliangrosshauser) [inline samples/Catalog/package.json]: Should we keep the Catalog version in sync with the package version?

Comment (simoarpe) [inline samples/Catalog/package.json]: That would be ideally a good idea but highly impracticable. Consider the case when the Catalog needs to be upgraded and the package does not. 
I would bump the corresponding version only when a change takes places, but we can discuss.

**Pull request merged**